pom.xml

```diff
@@ -12,7 +12,7 @@
   <version>1.36-SNAPSHOT</version>
 
   <properties>
-    <java.level>3</java.level>
+    <java.level>6</java.level>
```

Comment:
@daniel-beck See my comment here: https://github.com/jenkinsci/extras-executable-war/pull/11#discussion_r126609914

A while ago (1 year?) I have got a bug for raising it to Java 6 though